app/editor/components/MentionMenu.tsx

```diff
@@ -36,12 +38,14 @@ type Props = Omit<
 function MentionMenu({ search, ...rest }: Props) {
   const [items, setItems] = React.useState<MentionItem[]>([]);
   const { t } = useTranslation();
-  const { users, auth } = useStores();
+  const { documents, auth } = useStores();
+  const location = useLocation();
+  const documentId = parseDocumentSlug(location.pathname);
   const { view } = useEditor();
   const { data, request } = useRequest(
     React.useCallback(
-      () => users.fetchPage({ query: search, filter: "active" }),
-      [users, search]
+      () => documents.fetchUsers({ id: documentId, query: search }),
+      [documents, search]
```

Comment:
`documentId` should be in dependencies too, no?

Reply:
Yes!